### PR TITLE
Added 4 YouTube channels

### PR DESCRIPTION
**Pull request by Agrim-Sharma174**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,7 @@ If you're a YouTuber reading this and if you think you make beginner-friendly tu
 - [Programming With Mosh](https://www.youtube.com/programmingwithmosh) - Covers various fields/concepts of programming. 
 - [Code with Ania Kubów](https://www.youtube.com/AniaKub%C3%B3w) - Makes content around **Web Development**.
 - [Chris Courses](https://www.youtube.com/ChrisCourses) - Makes content around **Web Games**, written in HTML, CSS, JS.
+- [CodeWithHarry- Hindi](https://www.youtube.com/CodeWithHarry) - Teaches various programming languages and have many discussion videos around coding and tech.
```
Comment (KendallDoesCoding) [inline README.md:26]:
```suggestion
- [CodeWithHarry- Hindi](https://www.youtube.com/CodeWithHarry) - Makes content around various **programming languages** and has many discussion videos around **coding** and **tech**.
```

Comment (KendallDoesCoding) [inline README.md:26]:
```suggestion
- [CodeWithHarry- Hindi](https://www.youtube.com/CodeWithHarry) - Teaches various programming languages and have many discussion videos around coding and tech in **Hindi**.
```

File: README.md [hunk 2]
```diff
@@ -23,3 +23,7 @@ If you're a YouTuber reading this and if you think you make beginner-friendly tu
...
+- [ProgrammingWithHarry- English](https://www.youtube.com/programmingwithharry) - Same content in **English** language.
```
Comment (KendallDoesCoding) [inline README.md:27]:
```suggestion
- [ProgrammingWithHarry- English](https://www.youtube.com/programmingwithharry) - Makes content around various **programming languages** and has many discussion videos around **coding** and **tech** in **English**.
```

File: README.md [hunk 3]
```diff
@@ -23,3 +23,7 @@ If you're a YouTuber reading this and if you think you make beginner-friendly tu
...
+- [Apna College Hindi](https://www.youtube.com/ApnaCollegeOfficial) - Very famous among college students, teaches programming languages like **C**, **C++**, **JAVA** and **DSA**.
```
Comment (KendallDoesCoding) [inline README.md:28]:
```suggestion
- [Apna College Hindi](https://www.youtube.com/ApnaCollegeOfficial) - Very famous among college students, makes content around programming languages like **C**, **C++**, **JAVA** and **DSA** in Hindi.
```

File: README.md [hunk 4]
```diff
@@ -23,3 +23,7 @@ If you're a YouTuber reading this and if you think you make beginner-friendly tu
...
+- [CodeHelp - by Babbar Hindi](https://www.youtube.com/CodeHelpbyBabbar) - Teaches core subjects of Computer Science such as **DBMS**, **DSA**, **OS**, and programming languages like **Flutter**, **C++**, **Python**.
```
Comment (KendallDoesCoding) [inline README.md:29]:
```suggestion
- [CodeHelp - by Babbar Hindi](https://www.youtube.com/CodeHelpbyBabbar) - Makes content around core subjects of Computer Science such as **DBMS**, **DSA**, **OS**, and programming languages like **Flutter**, **C++**, **Python** in Hindi.
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by Agrim-Sharma174 — COMMENTED** [review]
Applied the changes

**Review by KendallDoesCoding — APPROVED** [review]
I think we should make Hindi bold too, but it's fine for now.

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @Agrim-Sharma174

**Pull request merged**